src/pug/blog/future_developments.pug

```diff
@@ -0,0 +1,159 @@
+      td: div
+        span Injection molded case and buttons, with superior quality.
+        p Sold in the shop as a "normal" controller.
+        p Ready to use: Pre-assembled, firmware pre-installed.
+        p Certified.
+
+  p DIY and PRO variants share features, electronics, and firmware. Only the external case is different.
+
+
+  hr
+  h3(style='margin-bottom:20px') Alpakka 2 enters design phase
+
+  div(style="display:inline")
+    div(style="display:inline-block; margin-bottom:10px; margin-right:20px")
+      iframe(
+        width="420"
+        height="232"
+        src="https://www.youtube.com/embed/ctiO1GMrKNk?si=kcaAxMubB_ngiuXB"
+        referrerpolicy="strict-origin-when-cross-origin"
+        allowfullscreen
+      )
+      div.caption Alpakka 2 reveal trailer
+    div(style="display:inline-block; margin-bottom:10px; margin-right:20px")
+      img(src='/static/blog/alpakka_2_teaser.jpg', width='420')
+      div.caption Alpakka 2 proposed design (not final)
+
+  p Alpakka 2 will be #[+ax('https://www.indiegogo.com/en/projects/input-labs/alpakka-2') funded via Indiegogo] for both DIY and PRO versions. Funding the development like this is necessary to pay for injection molds, CE and FCC cerfitications, and hopefully adding new members to the team to complement our expertise.
+  p The #[em Alpakka 2 vision] is to bring advanced gyro controls to #[em all PC players], and #[em reduce barriers] to its adoption.
+  p Some of the key features of the Alpakka 2 are:
+
+  table.manual.hdiv
+    tr
+      th Alpakka 2 feature (confirmed)
+      th But why?
+    tr
+      td Injection molded and pre-assembled (PRO version)
+      td Most requested feature in the community.
+    tr
+      td Quad-gyro
+      td Also a very requested feature. #[br] To get even more gyro performance, and reduce temperature-induced drift.
+    tr
+      td Display (OLED?)
+      td To preview the current mappings, and navigate config shortcuts. #[br] So profiles are less intimidating and self-explanatory for new users.
+    tr
+      td Smooth external case
+      td So people do not assume is non-ergonomic even before trying it.
+    tr
+      td Twin thumbsticks
+      td So the design is more familiar for people that are not fully sold on gyro gaming yet.
+    tr
+      td 2 Scrollwheels
+      td Having a scrollwheel has been surprisingly useful, and the community loves it. #[br] But what's better than a single scrollwheel? two of them!.
+
+  p Other features like #[em hall-effect] thumbsticks and #[em dual-action triggers] are to be evaluated and tested later, and are not yet confirmed. Please check the #[+ax('https://www.indiegogo.com/en/projects/input-labs/alpakka-2') Alpakka 2 Indiegogo page] for more details and features, and follow for a #[em free supporter pack] with stickers and keyring.
+  p We are still working on final adjustments for the actual campaign launch, please stay tuned, and share any feedback you may have!
+  +figure(
```

Comment:
This may require additional explanation. IMHO it should actually be removed from this report.